AuthenticationDemo/ExampleSessionService.swift

```diff
@@ -22,29 +23,28 @@ public struct ExampleUser {
     }
 }
 
-
 public final class ExampleSessionService: SessionServiceType {
     
-    private let (_currentUser, _currentUserObserver) = Signal<ExampleUser, NoError>.pipe()
-    public let currentUser: AnyProperty<ExampleUser?>
+    fileprivate let (_currentUser, _currentUserObserver) = Signal<ExampleUser, NoError>.pipe()
+    public let currentUser: Property<ExampleUser?>
     
-    private let _email: String
-    private let _password: String
-    private let _registeredAlready: Bool
+    fileprivate let _email: String
+    fileprivate let _password: String
+    fileprivate let _registeredAlready: Bool
     
     init(email: String, password: String) {
         _email = email
         _password = password
         _registeredAlready = false
-        currentUser = AnyProperty(initialValue: Optional.None, signal: _currentUser.map { $0 })
+        currentUser = Property(initial: Optional.none, then: _currentUser.map { $0 })
     }
     
     deinit {
         _currentUserObserver.sendCompleted()
     }
     
-    public func logIn(email: Email, password: String) -> SignalProducer<ExampleUser, SessionServiceError> {
-        let dispatchTime = dispatch_time(DISPATCH_TIME_NOW, (Int64)(2 * NSEC_PER_SEC))
+    public func logIn(_ email: Email, password: String) -> SignalProducer<ExampleUser, SessionServiceError> {
+        let dispatchTime = DispatchTime.now() + Double((Int64)(2 * NSEC_PER_SEC)) / Double(NSEC_PER_SEC)
```

Comment:
Hmm, this is interesting. Xcode changed this automatically for some reason. I think I'll delete the `/ Double(NSEC_PER_SEC)`